src/Orchard.Web/Modules/Orchard.Taxonomies/Projections/TermsFilterForms.cs

```diff
@@ -48,8 +48,13 @@ public class TermsFilterForms : IFormProvider {
                                 Id: "operator-is-all-of", Name: "Operator",
                                 Title: T("Is all of"), Value: "1"
                                 )
-                            )
-                        );
+                            ),
+                        _ExcludeChildren: Shape.Checkbox(
+                            Id: "ExcludeChildren", Name: "ExcludeChildren",
+                            Title: T("Automatically exclude children terms in filtering"),
+                            Value: "true"
```

Comment:
If this is `true` by default then isn't it a breaking change compared to the previous behaviour? If so then it should be false by default.

Reply:
The true value is correct, it is not the initial value of the checkbox, but is used by the mvc binding to assign the correct value to the property if the checkbox is checked.

Reply:
If I go to the TermsFilterForm, the checkbox will be ticked by default, right? So `ExcludeChildren.Value` will then be set to `true` if I save the form without changing anything. Then `bool.TryParse(context.State.ExcludeChildren?.Value, out bool excludeChildren)` will return `true`, and 

```
                    if (!excludeChildren)
                        allChildren.AddRange(_taxonomyService.GetChildren(term));
```

won't be executed, which was not the case before this PR

Reply:
Sorry if I join the discussion, but I think there was a misunderstanding. The `Shape.Checkbox(...)` code renders this kind of html 
`<input id="IncludeChildren" name="IncludeChildren" type="checkbox" value="true" />`
the attribute responsible of thicking the checkbox is the `checked` attribute not the `value` one. 
So this PR does not change the previous behaviour and it's not breaking. @sebastienros

Reply:
OK, thanks